scripts/cluster-upgrade.sh

```diff
@@ -0,0 +1,136 @@
+ #!/bin/bash
+
+OCP_VERSION_FROM=$1
+OCP_VERSION_TO=$2
+
+if [ "$OCP_VERSION_TO" == "" ];
```

Comment:
```suggestion
if [ -z $OCP_VERSION_TO ];
```
This is a bit more idiomatic Bash.